test/integration/model/notExist.test.js

```diff
@@ -41,18 +42,22 @@ describe(Support.getTestDialectTeaser('Model'), () => {
       await expect(this.Order.sum('amount', { where: { type: 'B' } })).to.eventually.be.equal(8);
       await expect(this.Order.max('amount', { where: { type: 'B' } })).to.eventually.be.equal(6);
       await expect(this.Order.min('amount', { where: { type: 'B' } })).to.eventually.be.equal(2);
-    });
 
-    it('should type not exist', async function() {
-      // DataTypes.INTEGER or DataTypes.BIGINT: previous version should use `.to.eventually.be.NaN`
       await expect(this.Order.sum('sequence', { where: { type: 'C' } })).to.eventually.be.equal(0);
       await expect(this.Order.max('sequence', { where: { type: 'C' } })).to.eventually.be.equal(0);
       await expect(this.Order.min('sequence', { where: { type: 'C' } })).to.eventually.be.equal(0);
-
-      // DataTypes.DECIMAL or DataTypes.FLOAT: previous and PR#13422 both use `to.eventually.be.equal(0)`
       await expect(this.Order.sum('amount', { where: { type: 'C' } })).to.eventually.be.equal(0);
       await expect(this.Order.max('amount', { where: { type: 'C' } })).to.eventually.be.equal(0);
       await expect(this.Order.min('amount', { where: { type: 'C' } })).to.eventually.be.equal(0);
     });
+
+    it('should type not exist', async function() {
```

Comment:
Could you please give that a more sensible title? Must have slipped through earlier :)

Reply:
I'm sorry, I'm non-native so I can't think of proper English...